uktrade/data-hub-frontend
# Fully revert Storybook back to 6.1.21

## Description of change

Somehow Storybook wasn't reverted fully back to 6.1.21 (when running it locally and in CI it was still using 6.2.9 even though that wasn't the installed version). 

<img width="375" alt="Screenshot 2021-06-28 at 16 12 51" src="https://user-images.githubusercontent.com/36161814/123660757-d5f92380-d82b-11eb-91a3-27ba36394d9e.png">

Reinstalling all the Storybook dependencies seems to have done the trick. I also installed `react-select` as without it Storybook would not build or run.

## Test instructions

Run `npm install`, `npm run storybook:build` and `npm run storybook`.

Storybook should now run locally and not show a WSOD, and the version running should be 6.1.21. If all the components start flashing then you need to remove `node_modules` and run the three commands again.

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
